pupil_src/shared_modules/version_utils.py

```diff
@@ -34,7 +34,18 @@ def get_tag_commit():
         return None
 
 
-def pupil_version():
+_Version = T.Union[packaging.version.LegacyVersion, packaging.version.Version]
```

Comment:
Why does this have a leading underscore?
It seems to be used from other files as well...

Reply:
There definitively some candidates... Here is a selection from `git tag --list`
```
v0.2.0
v0.2.0a1
v0.2.0a2
v0.3.0b1
v0.3.0b2
v0.3.0b3
v0.3.4bilateral
v0.6.18-windows
```

Reply:
@pfaion The leading underscore was temporary, to make searching through the codebase easier; it's gone now.

I also took the liberty to rename `_Version` to `SemanticVersion`. The reason is again to make search easier, especially since we're using the word `version` a lot for models, where the version is a single `int`. This name should signal that the value is more than just an `int`, but that it can be correctly compared to other instances. I'm open to suggestions of other names, but this one seems appropriate to me.

Since @papr touched on the need for `LegacyVersion` I'll resolve this comment; if you disagree, please reopen it and/or comment here.